services/scan/src/central_scanner_app.ts

```diff
@@ -142,27 +148,35 @@ export async function buildCentralScannerApp({
         ? request.files?.['package']?.[0]
         : undefined;
 
-      if (!file) {
-        response.status(400).json({
-          status: 'error',
-          errors: [
-            {
-              type: 'invalid-value',
-              message: 'expected file field to be named "package"',
-            },
-          ],
-        });
-        return;
-      }
+      try {
+        if (!file) {
+          response.status(400).json({
+            status: 'error',
+            errors: [
+              {
+                type: 'invalid-value',
+                message: 'expected file field to be named "package"',
+              },
+            ],
+          });
+          return;
+        }
 
-      const pkg = await readBallotPackageFromBuffer(await readFile(file.path));
+        const pkg = await readBallotPackageFromBuffer(
+          await readFile(file.path)
```

Comment:
I noticed you switched to `fs.readFile` in precinct scanner app, but not here. Should we do that here too?

Reply:
https://github.com/votingworks/vxsuite/pull/2707